examples/next/pages/ui/components/in-app-messaging/override-component/index.page.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback, useEffect } from 'react';
+import { Amplify, Notifications } from 'aws-amplify';
+import {
+  Button,
+  Flex,
+  Text,
+  useInAppMessaging,
+  useTheme,
+  withInAppMessaging,
+} from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+import config from './aws-exports';
+
+const { InAppMessaging } = Notifications;
+
+Amplify.configure(config);
+
+const CustomBannerMessage = (props) => {
+  const theme = useTheme();
+
+  return (
+    <Flex
+      alignItems="center"
+      borderRadius=".5rem"
+      position="absolute"
+      padding="2rem"
+      backgroundColor={theme.tokens.colors.teal[20]}
```

Comment:
```suggestion
      backgroundColor={teal.20}
```

So we do not need `useTheme();`